library/ssl_msg.c

```diff
@@ -754,12 +754,12 @@ int mbedtls_ssl_encrypt_buf( mbedtls_ssl_context *ssl,
         post_avail -= transform->maclen;
         auth_done++;
     }
-#endif /* MBEDTLS_SSL_SOME_MODES_USE_MAC */
+#endif /* MBEDTLS_SSL_SOME_SUITES_USE_MAC */
 
     /*
      * Encrypt
      */
-#if defined(MBEDTLS_ARC4_C) || defined(MBEDTLS_CIPHER_NULL_CIPHER)
+#if defined(MBEDTLS_SSL_SOME_MODES_USE_STREAM)
```

Comment:
typo, it should be MBEDTLS_SSL_SOME_SUITES_USE_STREAM
Please correct it in all occurence below.

Reply:
Fixed.